rust_src/src/windows.rs

```diff
@@ -672,6 +672,20 @@ pub fn window_live_p(object: Option<LispWindowRef>) -> bool {
     object.map_or(false, |m| m.is_live())
 }
 
+// Return new pixel size of window WINDOW.
+// WINDOW must be a valid window and defaults to the selected one.
+//
+// The new pixel size of WINDOW is the value set by the last call of
+// `set-window-new-pixel' for WINDOW.  If it is valid, it will be shortly
+// installed as WINDOW's pixel height (see `window-pixel-height') or pixel
+// width (see `window-pixel-width'). 
+
+#[lisp_fn]
+pub fn window_new_pixel(window: LispWindowLiveOrSelected) -> EmacsInt {
```

Comment:
You want to use `LispWindowValidOrSelected` here, `LispWindowLiveOrSelected` is for `decode_live_window`

Reply:
Changed.